CRITICAL - 2022-03-28 04:11:15 --> Value of type bool is not callable
#0 C:\xampp\htdocs\tripilar\system\Config\BaseService.php(248): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
#1 C:\xampp\htdocs\tripilar\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#2 C:\xampp\htdocs\tripilar\app\ThirdParty\myth-auth\src\Config\Services.php(84): CodeIgniter\Config\BaseService::getSharedInstance('activator', NULL)
#3 C:\xampp\htdocs\tripilar\system\Config\BaseService.php(248): Myth\Auth\Config\Services::activator()
#4 C:\xampp\htdocs\tripilar\system\Common.php(929): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#5 C:\xampp\htdocs\tripilar\app\ThirdParty\myth-auth\src\Controllers\AuthController.php(401): service('activator')
#6 C:\xampp\htdocs\tripilar\system\CodeIgniter.php(832): Myth\Auth\Controllers\AuthController->resendActivateAccount()
#7 C:\xampp\htdocs\tripilar\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#8 C:\xampp\htdocs\tripilar\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tripilar\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tripilar\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-03-28 04:12:42 --> Value of type bool is not callable
#0 C:\xampp\htdocs\tripilar\system\Config\BaseService.php(248): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
#1 C:\xampp\htdocs\tripilar\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#2 C:\xampp\htdocs\tripilar\app\ThirdParty\myth-auth\src\Config\Services.php(84): CodeIgniter\Config\BaseService::getSharedInstance('activator', NULL)
#3 C:\xampp\htdocs\tripilar\system\Config\BaseService.php(248): Myth\Auth\Config\Services::activator()
#4 C:\xampp\htdocs\tripilar\system\Common.php(929): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#5 C:\xampp\htdocs\tripilar\app\ThirdParty\myth-auth\src\Controllers\AuthController.php(401): service('activator')
#6 C:\xampp\htdocs\tripilar\system\CodeIgniter.php(832): Myth\Auth\Controllers\AuthController->resendActivateAccount()
#7 C:\xampp\htdocs\tripilar\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#8 C:\xampp\htdocs\tripilar\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tripilar\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tripilar\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-03-28 04:15:29 --> Call to a member function send() on null
#0 C:\xampp\htdocs\tripilar\system\CodeIgniter.php(832): Myth\Auth\Controllers\AuthController->resendActivateAccount()
#1 C:\xampp\htdocs\tripilar\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#2 C:\xampp\htdocs\tripilar\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\tripilar\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\tripilar\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
